pkg/idaes_examples/common/hda/ideal_VLE.py

```diff
@@ -919,7 +1004,7 @@ def rule_temp_dew(b):
                 return b.pressure * sum(b.mole_frac_comp[i] /
                                         b._p_sat_dewT[i]
                                         for i in ['toluene', 'benzene']) \
-                    - 1 == 0
+                    - 1*pyunits.Pa == 0
```

Comment:
This should not be required here - `_p_sat_dewT` should have units of pressure instead.